build-parent/pom.xml

```diff
@@ -89,13 +89,13 @@
         <opensearch.protocol>http</opensearch.protocol>
 
         <!-- Database images for JDBC/Reactive/Hibernate tests and devservices -->
-        <postgres.image>docker.io/postgres:14.2</postgres.image>
-        <mariadb.image>docker.io/mariadb:10.7</mariadb.image>
+        <postgres.image>docker.io/postgres:14</postgres.image>
+        <mariadb.image>docker.io/mariadb:10.6</mariadb.image>
```

Comment:
I think we shouldn't do that and regularly upgrade. People might want to use the latest features.

It's just a thought though, I won't fight for it.

Reply:
Another way to look at it is that dev mode should not use a newer db version than your production setup (which is more likely to be a longer supported version, so 10.6 in this case).

But I won't fight for it either. 😄